bots/orders/OrderFormNotion.ts

```diff
@@ -39,57 +43,67 @@ export default class OrderForm {
             });
 
             for (let i of orders.results) {
-                if (!("last_edited_time" in i) || !isFullPage(i)) {
+                if (!isFullPage(i) || !isRequisitionObjectPageResponse(i)) {
                     console.warn(`Order form checker: Received a partial response from Notion, skipping.`);
                     continue;
                 }
-
+                
                 const pageLastEditedTime = new Date(i.last_edited_time).getTime();
-                // Send a Discord message only after the first sync,
-                // and only if there is a new order form entry or an order form entry has changed.
-                if (!isFirstSync && (!this.idTimesMap[i.id] || pageLastEditedTime != this.idTimesMap[i.id])) {
-                                    // given the required title, product name, and subtotal are not filled out, skip until next sync
-                    if (!this.isCompletedSushiOrder(i)) {
-                        console.warn(`Order form checker: Received a partially filled out page ${i.id}, skipping until required title, product name, and subtotal are filled out`);
-                        continue;
-                    }
+                const orderStatus = i.properties["Status"].status?.name as OrderStatus;
+
+                if (isFirstSync) {
+                    this.syncLocalCache(i, pageLastEditedTime, orderStatus);
+                    continue; 
+                }
 
-                    const submitter = i.properties[orderFormProps.submitter];
-                    if (submitter != null && submitter?.type != "people") {
-                        console.warn(`Order form checker: Submitter not "people" type`);
+                if (this.requiresSyncing(i.id, pageLastEditedTime, orderStatus)) {
+                    let reqObj;
+                    try {
+                        reqObj = toRequisitionObject(i);
+                    } catch (error) {
+                        if (error instanceof TypeError) {
+                            console.warn(`Skipping order form entry ${i.id}: Not a valid RequisitionObject candidate.`, error);
```

Comment:
can we view console logs somewhere or is there something different we should be using for logging?

Reply:
afaik they're on the ktomega server machine (so no unless you're kevin, but still useful for debugging when running locally!)